x-pack/plugins/session_view/public/components/ProcessTree/helpers.ts

```diff
@@ -56,14 +56,17 @@ export const buildProcessTree = (
       if (backwardDirection) {
         orphans?.unshift(process);
       } else {
+        process.isOrphan = true;
         orphans?.push(process);
       }
     }
   });
 
   const newOrphans: Process[] = [];
 
+  // commenting this out as this is being handled by the flattenLeader function, but need tests
```

Comment:
If this doesn't break orphan handling and is now handled in the flatten call, then can we just remove this? I would at least test jumpToEvent feature to ensure orphans are being handled correctly.

Reply:
I'll make some tests with jumpToEvent